src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should not be a space before the colon in the modifier class (`.stars--4`). Please remove the space to comply with the style guide.